WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsMenuViewController.swift

```diff
@@ -45,10 +45,18 @@ private extension InPersonPaymentsMenuViewController {
     }
 
     var cardReadersSection: Section? {
-        let rows = [
+        var rows = [
             .orderCardReader,
-            .manageCardReader
+            .manageCardReader,
         ] + readerManualRows()
+
+        if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.consolidatedCardReaderManuals) {
+            // While not ready to be enabled, hide the individual rows, and show the consolidated version instead
+            let cardReaderManualRow = Row.cardReaderManuals
+            rows.removeLast(readerManualRows().count)
+            rows.append(cardReaderManualRow)
+        }
```

Comment:
Nice one for using a feature flag. This is great because it means you can merge this even though it still has UI issues, and you can break those out to future work 🥳 

This may be clearer:

```
let rows: [Row]

        if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.consolidatedCardReaderManuals) {
            rows = [
                .orderCardReader,
                .manageCardReader,
                .cardReaderManuals
            ]
        } else {
            rows = [
                .orderCardReader,
                .manageCardReader
            ] + readerManualRows()
        }
```

Reply:
Gotcha! 👍